src/main/java/seedu/multitasky/model/entry/EntryList.java

```diff
@@ -132,12 +132,6 @@ public void updateEntry(ReadOnlyEntry target, ReadOnlyEntry editedEntry)
 
         Entry entryToUpdate = internalList.get(index);
 
-        if (internalList.contains(editedEntry)) {
-            if (!isArchivedOrDeletedFloatingTask(editedEntry)) {
-                throw new DuplicateEntryException();
-            }
-        }
-
```

Comment:
@kevinLamKB 
This may result in merge conflict with my PR #255 
When you resolve the merge conflict, use the change over there cuz `DuplicateEntryException` needs to be thrown in some cases of edits, just that our existing tests haven't covered those cases I think. I'll add the test for that after this PR is merged.